src/main/java/com/klarna/hiverunner/HiveRunnerExtension.java

```diff
@@ -1,13 +1,13 @@
 /**
  * Copyright (C) 2013-2021 Klarna AB
  * Copyright (C) 2021 The HiveRunner Contributors
- *
+ * <p>
```

Comment:
I looked through some Apache projects to see how they handle this and most of them don't add the `<p>`. I think we shouldn't either, see https://github.com/apache/flink/blob/master/flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/AsyncDataStream.java for an example.